File: openapi-client/javascript-closure-angular/API/Client/SendMailFrom.js
goog.provide('API.Client.SendMail_from');

/**
 * An Email Contact
 * @record
 */
API.Client.SendMailFrom = function() {}

/**
 * The email address
 * @type {!string}
 * @export
 */
API.Client.SendMailFrom.prototype.email;

/**
 * Optional contact name
 * @type {!string}
 * @export
 */
API.Client.SendMailFrom.prototype.name;

File: openapi-client/scalaz/src/main/scala/org/openapitools/client/api/SendMailFrom.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime


import SendMailFrom._

case class SendMailFrom (
  /* The email address */
  email: String,
/* Optional contact name */
  name: Option[String])

object SendMailFrom {
  import DateTimeCodecs._

  implicit val SendMailFromCodecJson: CodecJson[SendMailFrom] = CodecJson.derive[SendMailFrom]
  implicit val SendMailFromDecoder: EntityDecoder[SendMailFrom] = jsonOf[SendMailFrom]
  implicit val SendMailFromEncoder: EntityEncoder[SendMailFrom] = jsonEncoderOf[SendMailFrom]
}
